infrastructure/proto/log_replication_metadata.proto

```diff
@@ -44,17 +47,44 @@ message ReplicationStatusVal {
     SNAPSHOT = 0;
     LOG_ENTRY = 1;
   }
-  SyncType type = 3;
+  SyncType syncType = 3;
+  SyncStatus status = 4;
```

Comment:
Planning to cancel the periodic task in `LogReplicationAckReader` explicitly or put a flag.
Otherwise, some special status like error or not started will get overwritten.